HelpSource/Classes/Function.schelp

```diff
@@ -434,19 +437,26 @@ code::
 ::
 
 
+
 argument::duration
 The duration of the function to plot in seconds. The default is 0.01.
-argument::server
-The Server on which to calculate the plot. This must be running on your local machine, but does not need to be the internal server. If nil the default server will be used.
+argument::target
+The target node on which to calculate the plot. See link::Reference/asTarget::. If it is a link::Classes/Server::, it must be running on your local machine, but does not need to be the internal server (see link::Classes/Buffer#-loadToFloatArray:: for details).
```

Comment:
Does the implementation really require a local server? Why? getToFloatArray should work or?

Also, it would be useful to explain what's happening (addAfter) with target and why. With Function:plot presumably target only matters if you are reading from a bus. I find 'on which to calculate the plot' a bit enigmatic for newbies.

Reply:
It uses `loadToFloatArray`, presumably for speed. In general, the class library seems to avoid `getToFloatArray`; I'm not sure why.

For the default 100ms plot duration, get... rather than load... will not be any slower -- might even be faster. For longer plots, load... is much faster.

Reply:
I think getToFloatArray was added later, and everyone, including me, ich copying the old one.

Reply:
IAC plotting on remote servers seems desirable.

Reply:
I think the `getToFloatArray` looks more complicated and insecure with the `wait` and `timeout` arguments. That might be the reason. Maybe we need the same thing as we have for SynthDef, something that tries to send first and if that fails, tries to load? All packed under a clean interface.